flink-doris-connector/src/main/java/org/apache/doris/flink/sink/writer/DorisWriter.java

```diff
@@ -130,6 +130,9 @@ private void abortLingeringTransactions(Collection<DorisWriterState> recoveredSt
              if(!state.getLabelPrefix().equals(labelPrefix)){
                  LOG.warn("Label prefix from previous execution {} has changed to {}.", state.getLabelPrefix(), executionOptions.getLabelPrefix());
              }
+             if (state.getDatabase() == null || state.getTable() == null) {
+                 continue;
```

Comment:
Thank you for your contribution. Would it be better to add some logs here to inform users that the historical state cannot be aborted?